docs/ru/query_language/queries.md

```diff
@@ -308,10 +308,10 @@ SELECT * FROM system.parts WHERE active
 ```bash
 $ ls -l /var/lib/clickhouse/data/test/visits/
 total 48
-drwxrwxrwx 2 clickhouse clickhouse 20480 мая   13 02:58 20140317_20140323_2_2_0
-drwxrwxrwx 2 clickhouse clickhouse 20480 мая   13 02:58 20140317_20140323_4_4_0
-drwxrwxrwx 2 clickhouse clickhouse  4096 мая   13 02:55 detached
--rw-rw-rw- 1 clickhouse clickhouse     2 мая   13 02:58 increment.txt
+drwxrwxrwx 2 clickhouse clickhouse 20480 may   13 02:58 20140317_20140323_2_2_0
+drwxrwxrwx 2 clickhouse clickhouse 20480 may   13 02:58 20140317_20140323_4_4_0
+drwxrwxrwx 2 clickhouse clickhouse  4096 may   13 02:55 detached
+-rw-rw-rw- 1 clickhouse clickhouse     2 may   13 02:58 increment.txt
 ```
```

Comment:
Enlish bash types month with first capital letter, and without extra spaces after. i.e. `May 13` not `may    13`.

Reply:
My fault. I'll fix.